spec/commands/v2/put_content_spec.rb

```diff
@@ -226,7 +226,7 @@
           described_class.call(payload)
           Commands::V2::Publish.call(content_id: content_id, update_type: "minor")
 
-          regex = /Content item user_facing_version=(\d+) and locale=en for content item=#{Regexp.quote(content_id)} conflicts/
+          regex = /Content item state=draft and locale=en for content item=#{Regexp.quote(content_id)} conflicts/
```

Comment:
I've had a quick look, and it looks like its just dependent on the order in which the validation occurs. I would guess that this has changed due to the changes in the PutContent command.

I'm not sure this is the best time, as I realise there is quite a lot of churn in the database, but I can rewrite the locking code such that it serialises the command processing, and fails on the lock version instead if you think that is necessary?